src/App.jsx

```diff
@@ -2,6 +2,8 @@ import { Route, Routes } from "react-router-dom";
 
 import { Footer, Navbar } from "./Components";
 
+import Courses from "./Components/Courses/Courses";
+
```

Comment:
It should be in Pages!

Reply:
Added courses in pages. Institutions was a component, hence left it as it is.